lib/core.ps1

```diff
@@ -1240,5 +1263,8 @@ $globaldir = $env:SCOOP_GLOBAL, (get_config GLOBAL_PATH), "$env:ProgramData\scoo
 #       Use at your own risk.
 $cachedir = $env:SCOOP_CACHE, (get_config CACHE_PATH), "$scoopdir\cache" | Where-Object { -not [String]::IsNullOrEmpty($_) } | Select-Object -First 1
 
+# OS information
+$WindowsBuild = [System.Environment]::OSVersion.Version.Build
```

Comment:
A bit late, but why has this to be a global variable?

Reply:
Aha, since there're some apps that have minimal Windows version requirement, I've planned to add a manifest field that shows minimal Windows version, and use this variable to check it.

But, this is not implemented now...